Bylaws.md

```diff
@@ -19,92 +19,116 @@
 ### Users
 
 <ol start="2">
-	<li>The most important participants in the project are people who use Apertium. The majority of our developers start out as users and guide their development efforts from the user's perspective</li>
-	<li>Users contribute to the Apertium project by providing feedback to developers in the form of bug reports and feature suggestions. As well, users participate in the Apertium community by helping other users.</li>
+	<li>The most important participants are people who use Apertium. The majority of our contributors start out as users and guide their development efforts from the user's perspective.</li>
+	<li>Users contribute to Apertium by providing feedback to developers in the form of bug reports and feature suggestions. As well, users participate in the Apertium community by helping other users.</li>
 </ol>
 
-### Developers
+### Contributors and contributions
 
 <ol start="4">
-	<li>Developers are  volunteers who are contributing time, code, documentation, or resources to Apertium. A developer that makes sustained, welcomed contributions to the project may be invited to become a committer, though the exact timing of such invitations depends on many factors</li>
+	<li>Contributors are volunteers who contribute time, code, documentation, or resources to Apertium. A contributor that makes sustained, welcomed contributions to Apertium may be invited to become a Voting Member, though the exact timing of such invitations is not specified.</li>
+	<li>Contributors who have administrative rights to a repository may exercise those rights almost fully, including granting rights to new contributors, making releases, or setting up automation. Potentially destructive or distruptive actions, such as deleting or renaming the repository or master branch, or breaking backwards compatibility, must be approved by a PMC member, who may decide the action requires a PMC vote.</li>
+	<li>Apertium recommends, but does not require, that contributors sign the Apertium Free/Open-Source Contributor Licence Agreement. However, contributions that are not licensed to Apertium in this way must be clearly marked with a GPLv2+ compatible licence, plus the author's name and email or other identifying information sufficient for a lawyer to reach the contributor.</li>
```

Comment:
There should be a clause either specifying that minors should not sign the CLA, or providing a procedure for a parent or legal guardian to sign it on behalf of the minor.